src/css/components/loading.scss

```diff
@@ -2,25 +2,66 @@
 $z-foreground: 200;
 
 .loading {
-  height: 50%;
+  bottom: 0;
+  height: 100%;
   left: 0;
-  position: relative;
+  position: absolute;
+  right: 0;
   top: 0;
   width: 100%;
   z-index: $z-foreground;
 }
 
+.loading-container {
```

Comment:
this is a container for the block loader if it is nested in something like a panel